src/vip-framework/checks/checkComptroller.ts

```diff
@@ -0,0 +1,81 @@
+import { expect } from "chai";
+import { Contract, Signer } from "ethers";
+import { ethers } from "hardhat";
+
+import COMPTROLLER_ABI from "../abi/comptroller.json";
+import { setMaxStalePeriodInBinanceOracle, setMaxStalePeriodInChainlinkOracle } from "../../utils";
+import { impersonateAccount, mine } from "@nomicfoundation/hardhat-network-helpers";
+import ERC20_ABI from "../abi/erc20.json";
+import VTOKEN_ABI from "../abi/vToken.json";
+import { parseUnits } from "ethers/lib/utils";
+
+const vETH_ADDRESS = "0xf508fcd89b8bd15579dc79a6827cb4686a3592c8"
+const vUSDT_ADDRESS = "0xfd5840cd36d94d7229439859c0112a4185bc0255"
+const ACCOUNT = "0x5a52E96BAcdaBb82fd05763E25335261B270Efcb";
+const CHAINLINK_ORACLE = "0x1B2103441A0A108daD8848D8F5d790e4D402921F";
+const USDT = "0x55d398326f99059fF775485246999027B3197955";
+const USDT_FEED = "0xB97Ad0E74fa7d920791E90258A6E2085088b4320";
+const ETH = "0x2170Ed0880ac9A755fd29B2688956BD959F933F8";
+const NORMAL_TIMELOCK = "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396";
+const ETH_FEED = "0x9ef1B8c0E4F7dc8bF5719Ea496883DC6401d5b2e";
+const XVS = "0x151B1e2635A717bcDc836ECd6FbB62B674FE3E1D";
+const COMPTROLLER = "0xfD36E2c2a6789Db23113685031d7F16329158384";
```

Comment:
Would it make sense to get these addresses from the json file of the npm package, and consider the network? This way, this check could be reused for testnet/mainnet